index.js

```diff
@@ -66,7 +66,26 @@ module.exports = () => {
 					enqueuedItems++;
 					debug(`Enqueued items increase | ${enqueuedItems} items`);
 					debug(`Handling message on topic ${topic}`);
-					await handler({ body: getBodyDecoded(brokeredMessage.body, brokeredMessage.applicationProperties.contentEncoding), applicationProperties: brokeredMessage.applicationProperties, properties: getProperties(brokeredMessage) });
+					const { applicationProperties } = brokeredMessage;
+					const { subscriptionName } = applicationProperties;
```

Comment:
I don't understand why we need this change.

Reply:
In order not to have to access the object's properties each time from different places, we de-structured the object and reused its properties, reducing the possibility of causing bus.